addons/l10n_nl/models/res_partner.py

```diff
@@ -0,0 +1,9 @@
+# coding: utf-8
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    l10n_nl_kvk = fields.Char(string='KVK-nummer')
```

Comment:
company_registry normally but we don't have that field on the partner (so, you are sure you need this for every customer and not just for the company?)

Reply:
What is company_registry ? This seems like a field where you can put anything, with no real definition.

In nl, there can be multiple "specific numbers" for the same company : oin or kvk. For the seller, kvk is mandatory, but for customer either kvk or oin is fine but you need one.

Reply:
KVK is like ondernemingsnummer in Belgium and is normally put in that field. But if we need it for the customer, we need this field.